cli/cmd/start/start.go

```diff
@@ -25,7 +27,20 @@ func StartCmd(ver version.Version) *cobra.Command {
 	startCmd := &cobra.Command{
 		Use:   "start",
 		Short: "Build project and start web app",
+		Args:  cobra.MaximumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) > 0 {
+				projectPath = args[0]
+				if strings.HasSuffix(projectPath, ".git") {
+					repoName, err := github.CloneRepo(projectPath)
```

Comment:
would be nice to add a console log mentioning Cloning Repo {URL}

Reply:
Adding this in the `CloneRepo` so that it will output log for both `rill init` and `rill start`